spec/controllers/admin_general_controller_spec.rb

```diff
@@ -150,4 +150,66 @@
     end
 
   end
+
+  describe 'GET #stats' do
+
+    it 'assigns the number of public bodies to the view' do
+      get :stats
+      expect(assigns[:public_body_count]).to eq PublicBody.count
+    end
+
+    it 'assigns the number of requests to the view' do
+      get :stats
+      expect(assigns[:info_request_count]).to eq InfoRequest.count
+    end
+
+    it 'assigns the number of users to the view' do
+      get :stats
+      expect(assigns[:user_count]).to eq User.count
+    end
+
+    it 'assigns the number of tracks to the view' do
+      get :stats
+      expect(assigns[:track_thing_count]).to eq TrackThing.count
+    end
+
+    it 'assigns the number of comments to the view' do
+      get :stats
+      expect(assigns[:comment_count]).to eq Comment.count
+    end
+
+    it 'assigns a Hash with grouped counts of requests by state to the view' do
+      InfoRequest.destroy_all
+
+      8.times { FactoryGirl.create(:successful_request) }
+      2.times { FactoryGirl.create(:info_request) }
+      FactoryGirl.create(:attention_requested_request)
+
+      get :stats
+      expect(assigns[:request_by_state]).to be_a(Hash)
+      expect(assigns[:request_by_state]).to include("successful" => 8)
+      expect(assigns[:request_by_state]).to include("waiting_response" => 2)
+      expect(assigns[:request_by_state]).to include("attention_requested" => 1)
```

Comment:
+1 to the extra tests!

Any reason this isn't a [single expect](http://www.betterspecs.org/#single), e.g:

```ruby
expect(assigns[:request_by_state]).
  to eq({ 'successful' => 8,
          'waiting_response' => 2,
          'attention_requested' => 1 })
```

Ditto for the spec below?